File: 2-Semestre/Taller de Programación/Personal/Arreglos/Ejercicio 8.cs
/*
Hacer un programa que lea 10 números, los ordene y muestre ordenados de menor a mayor.
*/

using System;

namespace Ejercicio_8
{
    public class Program
    {
        public static void Main(string[] args)
        {
            int[] A = new int[10];

            for (int i = 0; i < A.Length; i++)
            {
                Console.Write($"Ingrese un número #{i + 1}: ");
                int dato = Convert.ToInt32(Console.ReadLine());

                A[i] = dato;
            }

            Array.Sort(A); // Ordena el arreglo, por defecto de menor a mayor
            Console.WriteLine($"A: {string.Join(", ", A)}");
        }
    }
}

File: 1-Semestre/Desarrollo de algoritmos/Actividades/Arreglos (Semana 13)/Ejercicio 8.py
'''
Hacer un programa que lea 10 números, los ordene y muestre ordenados de menor a mayor.
'''

A = []
longitud = 10

for i in range(longitud):
    dato = int(input(f"Ingrese número (nº{i+1}): "))
    A.append(dato)

A.sort() # Se usa A.sort() para ordenar en este caso de menor a mayor.
print(A)